tests/DocumentationExamplesTest.php

```diff
@@ -1022,6 +1029,128 @@ public function testChangeStreamExample_1_4()
+        $cursor = $db->sales->aggregate([
+            ['$unwind' => '$items'],
+            ['$match' => ['items.fruit' => 'banana']],
+            [
+                '$group' => ['_id' => ['day' => ['$dayOfWeek' => '$date']],
+                'count' => ['$sum' => '$items.quantity']]
+            ],
+            [
+                '$project' => [
+                    'dayOfWeek' => '$_id.day',
+                    'numberSold' => '$count',
+                    '_id' => 0
+                ]
+            ],
+            ['$sort' => ['numberSold' => 1]]
+        ]);
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testAggregation_example_3()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+        $cursor = $db->sales->aggregate([
+            ['$unwind' => '$items'],
+            ['$group' => [
+                '_id' => ['day' => ['$dayOfWeek' => '$date']],
+                'items_sold' => ['$sum' => '$items.quantity'],
+                'revenue' => [
+                    '$sum' => [
+                        '$multiply' => ['$items.quantity', '$items.price']
+                    ]
+                ]
+            ]],
+            ['$project' => [
+                'day' => '$_id.day', 'revenue' => 1, 'items_sold' => 1,
+                'discount' => [
+                    '$cond' => [
+                        'if' => ['$lte' => ['$revenue', 250]],
+                        'then' => 25,
+                        'else' => 0
+                    ]
+                ]
+            ]]
+        ]);
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testAggregation_example_4()
+    {
+        if (version_compare($this->getServerVersion(), '3.6.0', '<')) {
+            $this->markTestSkipped('let of an object is not supported');
+        }
+
+        $db = new Database($this->manager, $this->getDatabaseName());
+        $cursor = $db->air_alliances->aggregate([
+            ['$lookup' => [
+                'from' => 'air_airlines',
+                'let' => ['constituents' => '$airlines'],
+                'pipeline' => [['$match' => [
+                        '$expr' => ['$in' => ['$name', '$constituents']]
+                ]]],
+                'as' => 'airlines'
+            ]],
+            ['$project' => [
+                '_id' => 0,
+                'name' => 1,
+                'airlines' => [
+                    '$filter' => [
+                        'input' => '$airlines',
+                        'as' => 'airline',
+                        'cond' => ['$eq' => ['$$airline.country', 'Canada']]
+                    ]
+                ]
+            ]]
+        ]);
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testRunCommand_example_1()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+        $cursor = $db->command(['buildInfo' => 1]);
```

Comment:
For illustrative purposes, I might suggest adding `$result = $cursor->toArray()[0];` on the subsequent line. This is what we currently do in https://docs.mongodb.com/php-library/current/tutorial/commands/.